code/game/objects/items/radio/detpack.dm

```diff
@@ -7,7 +7,7 @@
 	item_state = "plasticx"
 	flags_item = NOBLUDGEON
 	w_class = WEIGHT_CLASS_SMALL
-	layer = MOB_LAYER - 0.1
+	layer = MOB_LAYER - 2
```

Comment:
Would `BELOW_OBJ_LAYER` hit what you intend here instead?
We should try to avoid using such large magic numbers like this.

Reply:
Ill be honest, I have no fucking idea. Mob layer worked when I tested it with bodies and objects, and the plan was to have the det packs hidden under just about everything, because the only use I can see em having is traps atm.
If that works on hiding the packs under shit, then it works for me

Reply:
So i looked at the layers, and i think the best match is RESIN_STRUCTURE_LAYER
What you have right now is way to low it would go under piping and wiring and basically everything.

`XENO_HIDING_LAYER` is low enough, as it mimics a hiding runner (under tables, items etc)